kinto/storage.py

```diff
@@ -0,0 +1,305 @@
+from functools import wraps
+
+import requests
+import six
+from requests.exceptions import RequestException
+
+from cliquet import logger
+from cliquet.storage import (
+    StorageBase, exceptions, Filter,
+    DEFAULT_ID_FIELD, DEFAULT_MODIFIED_FIELD, DEFAULT_DELETED_FIELD)
+from cliquet.storage.memory import apply_sorting, get_unicity_rules
+from cliquet.utils import json, COMPARISON
+
+
+API_PREFIX = "/v0"
+
+
+FILTERS = {
+    COMPARISON.LT: 'lt_',
+    COMPARISON.MIN: 'min_',
+    COMPARISON.MAX: 'max_',
+    COMPARISON.NOT: 'not_',
+    COMPARISON.EQ: '',
```

Comment:
This is normal: when no filter is specified it means that it's ``field=test``. See [the kinto documentation](http://kinto.readthedocs.org/en/latest/api/cliquet/resource.html#filtering) about filtering.